lib/runtime_options.go

```diff
@@ -20,13 +20,22 @@
 
 package lib
 
-import null "gopkg.in/guregu/null.v3"
+import (
+	"fmt"
+	"strings"
+
+	"github.com/loadimpact/k6/js/compiler"
+	null "gopkg.in/guregu/null.v3"
+)
 
 // RuntimeOptions are settings passed onto the goja JS runtime
 type RuntimeOptions struct {
 	// Whether to pass the actual system environment variables to the JS runtime
 	IncludeSystemEnvVars null.Bool `json:"includeSystemEnvVars" envconfig:"K6_INCLUDE_SYSTEM_ENV_VARS"`
 
+	// JS compatibility mode: "latest" (Goja+Babel+core.js) or "es5" (plain Goja)
```

Comment:
this comment is outdated

Reply:
Thanks, thought I caught them all.

Fixed in 3c68d844.